src/main/kotlin/no/nav/tilleggsstonader/sak/brev/mellomlager/MellomlagringBrevService.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.tilleggsstonader.sak.brev.mellomlager
+
+import no.nav.tilleggsstonader.sak.infrastruktur.sikkerhet.SikkerhetContext
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class MellomlagringBrevService(
+    private val mellomlagerBrevRepository: MellomlagerBrevRepository,
+    private val mellomlagerFrittståendeBrevRepository: MellomlagerFrittståendeBrevRepository,
+) {
+
+    fun mellomLagreBrev(behandlingId: UUID, brevverdier: String, brevmal: String): UUID {
+        slettMellomlagringHvisFinnes(behandlingId)
```

Comment:
Kan en annen saksbehandler gå inn og f.eks. endre brevmal som vil slette brevet til en annen saksbehandler?
Og generelt, er brev knyttet bare til behandling, ikke sb? Gir dette mening for frittståendebrev?

Reply:
ja, en saksbehandler kan gå inn og slette en annens brev i en behandling. For frittstående brev er det unikt pr. saksbehandler.